admin/src/utils/parseTeamIdFromFigmaUrl.ts

```diff
@@ -18,13 +18,19 @@ export function parseTeamIdFromFigmaUrl(figmaUrl: string): string {
 			throw new FigmaURLError('Invalid Figma URL');
 		}
 		const pathname = url.pathname;
-		const teamIdRegex = /\/files\/\d+\/team\/(\d+)/;
-		const match = pathname.match(teamIdRegex);
-		if (!match || !match[1]) {
-			throw new FigmaURLError('Unable to parse team ID from Figma URL');
+		const orgTeamRegex = /\/files\/\d+\/team\/(\d+)/;
```

Comment:
Note: In the Figma UI, when you copy the URL on a paid plan, it matches this URL. However, if you are a starter plan, the default URL format is different. However, the URL does not necessarily indicate the user's plan. For instance, you can reformat a team ID for a starter team to match /files/\d+/team/<starter team ID>.